### ISSUE TITLE
Undefined index: xmlNamespace

### ISSUE THREAD

**Issue by phppirate** [intuit/QuickBooks-V3-PHP-SDK#91]
Trying to create a customer and I keep getting this error when I try to add it using the DataService->Add() method.

**Comment by hlu2** [comment]
@phppirate 
I believe it is a class import issue, what is the actual exception you get?

**Comment by phppirate** [comment]
I have already figured this out... I was trying to use `new Customer` when you apparently need to use `Customer::create()`.

There was no exception thrown. It just failed with the error `Undefined index: xmlNamespace `

**Comment by amazingwebdev** [comment]
@hlu2 I am still getting that issue when adding a new quickbook obj such as Bill, BillPayment, Invoice.
For more information, I've created bill object first. (Bill::create($request))
I've encoded the obj to json and saved to db.
After some events, I was trying to add the quickbook obj to quickbook ($this->dataService->add($qbObj); the $qbObj is decoded quickbook obj from the table)
I think it should work, but getting the same issue when adding quickbook obj.
